yt_dlp/extractor/slideslive.py

```diff
@@ -139,6 +277,51 @@ def _real_extract(self, url):
         assert service_name in ('url', 'yoda', 'vimeo', 'youtube')
         service_id = player_info['service_id']
 
+        slides, slides_xml = None, None
+        slide_url_template = self._JPG_SLIDE_TMPL
+        chapters, thumbnails = [], []
+        if url_or_none(player_info.get('thumbnail')):
+            thumbnails.append({'url': player_info['thumbnail']})
+
+        if player_info.get('slides_json_url'):
+            slides = traverse_obj(self._download_json(
+                player_info['slides_json_url'], video_id, fatal=False,
+                note='Downloading slides JSON', errnote=False), 'slides', expected_type=list)
+        if slides:
+            if re.match(r'.+(/v[45]/).+', player_info['slides_json_url']):
+                slide_url_template = self._PNG_SLIDE_TMPL
+            for i in range(len(slides)):
+                slide_name = traverse_obj(slides, (i, 'image', 'name'))
+                slide_id = str(i + 1)
+                if slide_name:
+                    thumbnails.append({
+                        'id': slide_id,
+                        'url': slide_url_template % (video_id, slide_name),
+                    })
+                chapters.append({
+                    'title': slide_id,
```

Comment:
not `slide_name`? If there's no chapter title, core code will add `<Untitled chapter i>`

https://github.com/yt-dlp/yt-dlp/blob/c73355510629e3eda5a79d4e2876a35316ca6ed2/yt_dlp/YoutubeDL.py#L2517-L2525

Reply:
the `slide_name`s are random and unhelpful like `EHjb__1649109731`, so the core code's default is actually preferable in the off-chance that the `slide_id` value doesn't exist